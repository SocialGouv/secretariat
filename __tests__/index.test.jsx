import { render } from "@testing-library/react"
import { vi, it, expect } from "vitest"

import Index from "@/pages/index"

vi.mock("next-auth/react", () => ({
  useSession: () => ({
    data: null,
    status: "unauthenticated",
  }),
}))

it("renders home page", () => {
  const { container } = render(<Index />)
  expect(container).toMatchSnapshot()
})
