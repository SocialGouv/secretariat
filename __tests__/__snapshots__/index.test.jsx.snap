// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders home page 1`] = `
<div>
  <main
    class="flex flex-col pt-12 gap-12"
  >
    <div
      class="home"
    >
      <div
        class="alert warning"
      >
        <div
          class="icon"
        >
          <i
            class="ri-alert-fill"
          />
        </div>
        <div
          class="body"
        >
          <div
            class="title"
          >
            Version Beta
          </div>
          <div
            class="message"
          >
            Cette application est toujours en cours de développement. N'hésitez pas à signaler tout dysfonctionnement au support technique de la Fabrique Numérique.
          </div>
        </div>
      </div>
      <div
        class="jumbo"
      >
        <div
          class="text"
        >
          <p>
            Bienvenue au
          </p>
          <h2>
            Secrétariat de la Fabrique Numérique des ministères sociaux
          </h2>
          <p>
            Secrétariat est une solution numérique permettant aux membres des Startups d'Etat de gérer leurs accès aux services de la Fabrique Numérique des ministères sociaux.
          </p>
        </div>
        <div
          class="image"
        >
          <span
            style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
          >
            <span
              style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%271%27%20height=%271%27/%3e"
                style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
              />
            </span>
            <img
              alt="home image"
              data-nimg="intrinsic"
              decoding="async"
              src="/public/images/home.svg"
              srcset="/public/images/home.svg 1x"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
            />
          </span>
        </div>
      </div>
      <div
        class="tiles"
      >
        <div
          class="card"
        >
          <h3
            class="title"
          >
            Effectuez une demande d'embarquement
          </h3>
          <p>
            Si vous arrivez aux ministères sociaux et que vous avez besoins de créer vos comptes afin d'accéder aux services de la Fabrique Numérique, vous pouvez effectuer une demande d'embarquement en cliquant sur le bouton ci-dessous.
          </p>
          <a
            class="btn primary"
            href="/onboarding"
          >
            Demande d'embarquement
          </a>
        </div>
        <div
          class="card"
        >
          <h3
            class="title"
          >
            Administrer les comptes utilisateurs
          </h3>
          <p>
            Si vous souhaitez effectuer une revue des comptes des utilisateurs des services de la Fabrique Numérique des ministères sociaux, vous pouvez accèder aux outils de gestion en cliquant sur le bouton ci-dessous.
          </p>
          <a
            class="btn primary"
            href="/accounts"
          >
            Gestion des comptes
          </a>
        </div>
      </div>
    </div>
  </main>
</div>
`;
